chainer/optimizer.py

```diff
@@ -530,10 +530,11 @@ def use_cleargrads(self, use=True):
                 (`zerograds` is used).
 
         .. note::
-           Note that :meth:`update` calls :meth:`~Link.zerograds` by default
-           for backward compatibility. It is recommended to call this method
-           before first call of `update` because `cleargrads` is more
-           efficient than `zerograds`.
+           Note that :meth:`update` calls :meth:`~Link.cleargrads` by default.
+           :meth:`~Link.cleargrads` is more efficient than
+           :meth:`~Link.cleargrads`, so one do not have to call
+           :meth:`use_cleargrads`. This method remains for backward
+           compatibility.
```

Comment:
1. `is more efficient than Link.cleargrads` -> `... than Link.zerograds`
2.  `one do not` -> `one does not`
3. I think this method can be marked as deprecated, like `link.zerograds()`.